packages/react/package.json

```diff
@@ -38,7 +38,7 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "yarn run build:ts && yarn run build:catalog",
-    "build:ts": "tsup --minify --keep-names",
+    "build:ts": "tsup --minify",
```

Comment:
`--keep-names` was causing a build error for the map component. Can anyone provide a use-case for needing this flag?